vnext/Shared/Shared.vcxitems.filters

```diff
@@ -192,6 +294,78 @@
     <Filter Include="Source Files\Networking">
       <UniqueIdentifier>{71196e04-aca0-48d6-99e4-f418c957b50f}</UniqueIdentifier>
     </Filter>
+    <Filter Include="Header Files\Fabric">
+      <UniqueIdentifier>{e045612e-bca3-4844-bce8-e75f83e89e05}</UniqueIdentifier>
+    </Filter>
+    <Filter Include="Header Files\Fabric\codegen">
+      <UniqueIdentifier>{c6241404-615b-4338-8955-e9282c440712}</UniqueIdentifier>
+    </Filter>
+    <Filter Include="Header Files\Fabric\codegen\react">
+      <UniqueIdentifier>{00b927dc-e868-4e01-80a7-03adbd7ef5af}</UniqueIdentifier>
+    </Filter>
+    <Filter Include="Header Files\Fabric\codegen\react\components">
+      <UniqueIdentifier>{81771765-0676-4cf8-baab-3656dd4d761f}</UniqueIdentifier>
+    </Filter>
+    <Filter Include="Header Files\Fabric\codegen\react\components\rnwcore">
+      <UniqueIdentifier>{b84ea036-35e5-4c33-b029-14b35dcbb5ce}</UniqueIdentifier>
+    </Filter>
+    <Filter Include="Header Files\Fabric\platform">
+      <UniqueIdentifier>{0198c3a8-dcd1-4f17-aa80-769fa8df762f}</UniqueIdentifier>
+    </Filter>
+    <Filter Include="Header Files\Fabric\platform\react">
+      <UniqueIdentifier>{9bbd1367-8911-4f05-8c3a-b1b82899413d}</UniqueIdentifier>
+    </Filter>
+    <Filter Include="Header Files\Fabric\platform\react\renderer">
+      <UniqueIdentifier>{f34dccd6-7da9-46e6-b5e2-67f02690ff1d}</UniqueIdentifier>
+    </Filter>
+    <Filter Include="Header Files\Fabric\platform\react\renderer\graphics">
```

Comment:
>Header Files [](http://example.com/codeflow?start=21&length=12)

Not for this PR, but I find separation files into headers vs source in the VS tree to be quite inconvenient. I would rather see them next to each other as we do it in the Microsoft.ReactNative project.

Reply:
I agree. I just added them there because that's what Shared currently does.

Reply:
+1